operator/gitops/argocd/pipeline-service/metrics-exporter/kustomization.yaml

```diff
@@ -7,9 +7,9 @@ resources:
   - service.yaml
 
 images:
-  - name: quay.io/redhat-appstudio/user-workload
-    newName: quay.io/redhat-appstudio/user-workload
-    newTag: efc47cda884fc436211a057291d7493de0e6d6ce
+  - name: quay.io/redhat-appstudio/pipeline-service-exporter
```

Comment:
for this to work you need to update https://github.com/openshift-pipelines/pipeline-service/blob/main/operator/gitops/argocd/pipeline-service/metrics-exporter/deployment.yaml#L20 so that the image ref is `pipeline-service-exporter` otherwise it won't be able to match

Reply:
Nice catch. Updated.